Application/EdFi.Ods.AdminApi.AdminConsole/Infrastructure/DataAccess/Artifacts/Security/PgSql/20241217230946_AdminConsoleClaimset.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EdFi.Ods.AdminApi.AdminConsole.Infrastructure.DataAccess.Artifacts.Security.PgSql
+{
+    /// <inheritdoc />
+    public partial class AdminConsoleClaimset : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            var sqlFile = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Infrastructure/DataAccess/Artifacts/Security/PgSql/SQL/AdminConsoleClaimsetUp.sql");
```

Comment:
Does this mean that the SQL file must be deployed as a text file via the NuGet package? Is there any security risk associated with this?

Reply:
Yes, it's going to be part of the NuGet package as plain text.